lib/data/pokemons.dart

```diff
@@ -58,3 +58,47 @@ Color getPokemonColor(String typeOfPokemon) {
       return AppColors.lightBlue;
   }
 }
+
+// Taken from Bulbapedia - Type color templates
+Color getPokemonColor2(String label){
```

Comment:
Is it okay if I specify the color based on type? like FireColor or say Orange

Reply:
And when you say "new", you mean a unique color isn't specified for the type right? because we need different colors for each type to make this feature work

Reply:
> Is it okay if I specify the color based on type? like FireColor, or should I still to normal color names like Orange
You should use `orange`, so when the color is used in another place but not pokemon type, it should not make any confusion.

> And when you say "new", you mean a unique color isn't specified for the type right? because we need different colors for each type to make this feature work
Yes. Currently, the same color is used in multiple types. If you can add some new colors so that each type is using their unique color, that's cool. And make sure it looks good in the `Pokedex` screen too. I will prefer the color which is matched with the app's color system, it doesn't need to be exactly like Bulbapedia. 

Some types you should keep the old color are: Grass, Fire, Water, Electric, Poison, Ground, Dark. The other is up to you.

Reply:
> Some types you should keep the old color are: Grass, Fire, Water, Electric, Poison, Ground, Dark. The other is up to you.

There are some new colors that really suit Grass, Poison and Ground better. Let me make a tentative change and share the screenshots with you...

Reply:
@jitinder okay

Reply:
![pokedex1](https://user-images.githubusercontent.com/9639312/85207507-e3411f80-b346-11ea-9e4c-564eee0d0853.jpg)
![pokedex2](https://user-images.githubusercontent.com/9639312/85207508-e3d9b600-b346-11ea-90d7-5892dd52881c.jpg)
![pokedex3](https://user-images.githubusercontent.com/9639312/85207509-e4724c80-b346-11ea-9f2a-5f1b370f9195.jpg)

The changes in color are:

Grass: lightTeal -> lightGreen (bulba)
Poison: lightPurple -> purple
Ground: lightBrown -> brown

Reply:
It looks good! And also I think the ground type should be darker

Reply:
Ill replace ground with the Rock color, should be good enough

Reply:
I'll add a darker shade for `ground`